TNW Copy - Milestone 1 - Review

zilton7 opened the pull request:
Anded HTML and Styling for 4 sections for Microverse's Project 2 Milestone 1

zilton7 committed:
Add TWC title

zilton7 committed:
Add header HTML

zilton7 committed:
Add cover section and its articles HTML

zilton7 committed:
Add footer HTML; add nav .social and nav .links HTML

zilton7 committed:
Add styling for header logo and nav; add logo image and search and menu pngs to images dir

zilton7 committed:
Add cover section article styling and add cover article images to image dir

zilton7 committed:
Style footer and copyright session, add fontawesome link inside head

zilton7 committed:
Add latest news article images to images dir

zilton7 committed:
Remove unecessary latest news article images from images dir

zilton7 committed:
Add HTML for funding section; add images to images dir

zilton7 committed:
Add funding section header h3 and nav styling

zilton7 committed:
Add styling for .card and .cards

zilton7 committed:
Style card; add padding and style border top for description

zilton7 committed:
Style funding cards

zilton7 committed:
Add absolute position to funding title image

zilton7 committed:
Finish mobile styling

zilton7 committed:
Fix mobile styling by adding 62.5% font size to html tag

zilton7 committed:
Update index.html

fix linters issues

zilton7 committed:
Start styling tablet view

zilton7 committed:
Fix alignation for 3 funding areas make its title and body aligned together

zilton7 committed:
Make top nav and yeallow add fixed

zilton7 committed:
Done styling for tablet

zilton7 committed:
Finish up Desktop style

zilton7 committed:
Fix linter issues in css file

zilton7 committed:
Merge branch 'review-milestone-1' of github.com:zilton7/twc-copy into review-milestone-1

trekab commented on index.html:
```diff
@@ -4,15 +4,242 @@
+      <article class="cover-story"
+        style="background: linear-gradient(rgba(0, 0, 0, 0.3), rgba(0, 0, 0, 1)), url('./images/Opal-796x417.jpg') ;">
+        <div class="cover-header">
+          <h2 class="cover-title">Opal cuts off your apps' internet access to help you stop doomscrolling</h2>
```
It is highly recommended to avoid making use of inline styles because they are at the lowest level of the cascade. This makes them override other styles. [Here](https://speckyboy.com/good-bad-css-practices/) is a great article about good and bad CSS practices.

trekab reviewed (changes requested):
### Changes Required! :recycle: 
Hi @zilton7! Well done so far! Your implementation of the breakpoints following the mobile-first approach is commendable! :+1:  However, there are some details for you to work on to complete this milestone. These include: 
- [x] Add a link to your logo (`a` tag) since it is a link in the [original site](https://perma.cc/M5ZV-Q2D6) as well.
- [x] The hamburger button and the search button should also be links.
- [x] For the tablet screens, the Menu and submenu should appear on the header, and the hamburger button should disappear.
- [x] The TNW logo should disappear on desktop screens.
- [x] For the top 3 articles, the TWN logo appears over the first article, at the top left corner for desktop screen sizes.
- [x] There should be a red overlay with a gradient in the top half of the first article for desktop screen sizes.
- [x] The date should appear over the first article, at the top right corner on desktop screen sizes.
- [x] The Latest Funding section's header should be a link.
- [x] Under the Latest Funding section, each article's elements i.e. the title, logo image, the two category items, and the text at the bottom are all clickable links. Consider making proper use of the `a` tag for these sections.
- [x] The Latest Funding section's nav menu should be to the left of the main title, and the options aligned in a row, separated by `|`
- [x] For the footer, your social media icons appear to be way bigger than the ones on the original site and they tend to change in size on mobile screens. Please note that the footer should not change through all the breakpoints.

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink: 
__Treasure Kabareebe__
[Portfolio](https://trekab.github.io/portfolio/) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

zilton7 commented:
Thank you, that's really in-depth review, will star working on the issues tomorrow morning 😉

zilton7 committed:
Fix issue for linter css file

trekab commented:
__Friendly reminder:__ Please don't close or merge this pull request until it has been approved. You should maintain this same pull request for all code reviews for this milestone!

zilton7 commented:
Yes i know it, i closed it accidentally, reopened right away. Thanks 😊

zilton7 committed:
Fix linter issues for css file

zilton7 committed:
Add z-index:1 for header nav and yellow ad

zilton7 committed:
Update README file and fix linter issue in css file

zilton7 committed:
Turn header logo, hamburger and search buttons into the links

zilton7 committed:
Add main menu to header for tablets

zilton7 committed:
Update style.css

fix linter issues with a multi-line declaration   declaration-colon-newline-after, which is auto formatted by vscode

zilton7 committed:
Style main-menu navs and items

zilton7 committed:
Update style.css

zilton7 committed:
Update style.css

zilton7 committed:
Update style.css

zilton7 committed:
Update style.css

zilton7 committed:
Update style.css

zilton7 committed:
Update style.css

zilton7 committed:
Update style.css

zilton7 committed:
Update style.css

zilton7 commented:
All issues fixed, but when I try to resubmit form i get 'Project You already have an open code review request for Project 2: Building with Responsive Design - Milestone 1'

zilton7 committed:
Add red gradient and logo on first cover story image for desktop

zilton7 committed:
Make stuff clickable in Funding section

zilton7 committed:
Fix footer too look the same for all screens

zilton7 committed:
Add date to cover image for Desktop

zilton7 committed:
Make cover story image into cover for desktop

zilton7 committed:
Fix linter issue w/ funding section seperators

zilton7 committed:
Fix linter issues

zilton7 committed:
Fix issue with 4th card visibility in funding section

zilton7 committed:
Fix linter issues

zilton7 committed:
Fix linter issues

zilton7 committed:
Fix linter issues
